packages/ui/package.json

```diff
@@ -64,7 +63,8 @@
     "autoprefixer": "^10.3.1",
     "glob": "^10.3.10",
     "postcss": "^8.4.31",
-    "sass": "^1.35.2"
+    "sass": "^1.35.2",
+    "style-dictionary": "3.9.1"
```

Comment:
We still use it as a dev dependency because we use it to check to make sure all the references in our theme resolve properly.